snap_format.pm

```diff
@@ -4,414 +4,370 @@ sub write_txt {
   my %h = @_;
 
 # construct the snapshot page
-my $s = sprintf "UTC %s f_ACE %.2e F_CRM %.2e Kp %.1f R km%7s%s\n",${$h{UTC}}[1],${$h{FLUXACE}}[1],${$h{CRM}}[1],${$h{KP}}[1],${$h{EPHEM_ALT}}[1],${$h{EPHEM_LEG}}[1];
+my $s = sprintf "UTC %s   f_ACE %.2e F_CRM %.2e Kp %.1f R km%7s%s\n",
+                ${$h{UTC}}[1], ${$h{FLUXACE}}[1], ${$h{CRM}}[1], ${$h{KP}}[1], ${$h{EPHEM_ALT}}[1], ${$h{EPHEM_LEG}}[1];
 
-$s .= sprintf "OBT %s  CTUVCDU %8d  OBC s/w %s  %s_%-4s   CPEstat %s\n",
-    ${$h{OBT}}[1],${$h{CCSDSVCD}}[1],${$h{CONLOFP}}[1],${$h{CCSDSTMF}}[1],${$h{COTLRDSF}}[1],${$h{AOCPESTL}}[1];
+$s .= sprintf "OBT %s  CTUVCDU %8d  OBC s/w %s  %s_%-4s   CPEstat   %s\n",
+              ${$h{OBT}}[1], ${$h{CCSDSVCD}}[1], ${$h{CONLOFP}}[1], ${$h{CCSDSTMF}}[1], ${$h{COTLRDSF}}[1], ${$h{AOCPESTL}}[1];
 
-$s .= sprintf "OBT %17.2f  ONLVCDU %8d  OBC Errs%4d\n",${$h{OBT}}[0],${$h{OFLVCDCT}}[1],${$h{COERRCN}}[1];
+$s .= sprintf "OBT %17.2f  ONLVCDU %8d  OBC Errs%4d\n",
+              ${$h{OBT}}[0], ${$h{OFLVCDCT}}[1], ${$h{COERRCN}}[1];
 
-$s .= sprintf "                                                     OBSID  %5d  EPState %4s\n",
-    ${$h{COBSRQID}}[1], ${$h{EPSTATE}}[1];
+$s .= sprintf "%52 sOBSID  %5d  EPState   %4s\n",
```

Comment:
is it okay for the %52 and s to be separated?

Reply:
No.. Thanks! Good catch.